OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+OpacitySemanticTokens.swift

```diff
@@ -1,8 +1,4 @@
 //
-// Software Name: OUDS iOS
```

Comment:
I've spotted this issue. I'll regenerate the PR with the fix. Please still check the reste of the file :)